src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -41,6 +42,7 @@ import {
     isScatterPlot,
     isBubbleChart,
     isComboChart,
+    isInvertedChartType,
```

Comment:
there are multiple exceptions for the bar chart mainly because it behaves as inverted (rotated) column chart. The same applies to bullet chart so now we have a common function to detect them.